build/build.go

```diff
@@ -12,9 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tools
```

Comment:
Any reason to not have the build directory live in `internal`?

Reply:
Why I decided to NOT put it in `internal`:

1. Less nesting
2. In fact it does not expose anything as `package main` cannot be imported.
3. "Follows" https://github.com/golang-standards/project-layout 
4. [Nuke](https://nuke.build/index.html) which I discovered after creating `goyek` also contains by default creates build pipeline source code in `build` directory
5. K8s also has stuff related with building in `build` directory: https://github.com/kubernetes/kubernetes/tree/master/build

The OTel Go repos are the first place where I saw the `tools` package inside the `internal` directory.